packages/realm_dart/lib/src/app.dart

```diff
@@ -219,16 +219,17 @@ class App implements Finalizable {
 
   /// Returns the current value of the base URL used to communicate with the server.
```

Comment:
Just added this to the JS API docs as well (taken from Core). May be valuable to point out the extra information.
```suggestion
  /// Returns the current value of the base URL used to communicate with the server.
  ///
  /// If an [updateBaseUrl] operation is currently in progress, this value will not
  /// be updated with the new value until that operation has completed.
```